src/TestFramework/TestLogger.php

```diff
@@ -0,0 +1,190 @@
+<?php
+
+namespace PaymentGatewayLogger\TestFramework;
```

Comment:
this file would be better placed in tests/Framework instead of src/TestFramework. omnipay-bluesnap does it better than omnipay-vindicia cuz I got better at composer. generally you don't want to force ppl to install your testing framework to use the package.